src/debugger.ts

```diff
@@ -0,0 +1,105 @@
+import {CMakeCache} from '@cmt/cache';
+
+
+export namespace Debugger {
```

Comment:
Namespaces are a strange beast. I used them in `vscode-gitflow`, but I didn't find them very useful and just made things cluttered. I'd prefer to just use regular imports instead.